### ISSUE TITLE
Can't see the generated Inpaint image in webui. And no response after clicking Generate button

### ISSUE THREAD

**Issue by NobodyJoe** [TheLastBen/fast-stable-diffusion#759]
The images created by Inpaint are not displayed in Webui, but the generated results can be found in the output file in gdrive. And after that, the webui does not respond to clicking on generate when replacing the image..

**Comment by TheLastBen** [comment]
I'll check it out

**Comment by oohohoo** [comment]
UI needs to be refreshed after every generation. sometimes it gets stuck in the middle, but the results are visible in Google Drive after it finishes :)

**Comment by oohohoo** [comment]
i use old version 1.5 :)

**Comment by TheLastBen** [comment]
yes but the resolution of the image ?

**Comment by oohohoo** [comment]
512

**Comment by TheLastBen** [comment]
it might be the server being slow and causing the freeze, but it's only temporary.

**Comment by oohohoo** [comment]
the same thing happens on img2img when I put a higher resolution, for example 1088x1408. it generates one with no problem, and after that I need to refresh the UI. If I use the x/y plot, then it will work without problems until the end, no matter how many creations I put in the x/y plot, but after it ends I need to refresh the UI before the next creation. I noticed this for the first time yesterday, before that it was working normally. :)

**Comment by TheLastBen** [comment]
yes it was working normally because the tunnel server was working fine, but sometimes it is slow and causes gradio to freeze

**Comment by oohohoo** [comment]
I understand now. Thank you 👍
